samples/CMakeLists.txt

```diff
@@ -130,17 +130,18 @@ macro(add_sample)
     )
 
     foreach(CONFIG ${OPENCL_SAMPLE_CONFIGS})
-        install(TARGETS ${OPENCL_SAMPLE_TARGET} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR})
-        install(FILES ${OPENCL_SAMPLE_KERNELS} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR})
-        install(FILES ${OPENCL_SAMPLE_SHADERS} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR})
+        install(TARGETS ${OPENCL_SAMPLE_TARGET} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR}/${CONFIG})
+        install(FILES ${OPENCL_SAMPLE_KERNELS} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR}/${CONFIG})
+        install(FILES ${OPENCL_SAMPLE_SHADERS} CONFIGURATIONS ${CONFIG} DESTINATION ${CMAKE_INSTALL_BINDIR}/${CONFIG})
```

Comment:
This means that instead of the samples being installed to `<install_prefix>/install/bin` they will be installed to `<install_prefix>/install/bin/<config>`, so something like `/path/to/sdk/install/bin/Release`.  Was this intended?  I'm not opposed to this change, but it is different than what we had previously.

Reply:
It happened when adding the tests jobs to CI that in some environments the execution of the tests failed when configuring and building for Debug and Release (so first we configured for Debug and Release, then we built for Debug and Release and then we executed), and separating the install of the executables was solving it. Currently I'm not able to reproduce the old buggy behaviour, so probably some of the other changes of this PR triggered the fix for it. We can undo this change then IMHO.

Reply:
If the older behavior still works I think I'd slightly lean towards keeping the same installation directories, but this is something the working group should decide.

Reply:
Okay, let's see it then in the next WG meeting